api/server/utils/math.js

```diff
@@ -32,9 +32,9 @@ function math(str, fallbackValue) {
     throw new Error('Invalid characters in string');
   }
 
-  const value = eval(str);
+  const value = parseInt(str, 10);
 
-  if (typeof value !== 'number') {
+  if (isNaN(value)) {
```

Comment:
I assume that eval is correct in this place since there is a regex above checking for a valid string containing only digits and mathematical characters, i.e. numerical equation.

Reply:
The parseInt will turn it into a base 10 number and [isNan](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN) will check if it succeeded.